.pre-commit-config.yaml

```diff
@@ -32,6 +32,8 @@ repos:
       hooks:
       - id: flake8
         language_version: python3
+        additional_dependencies:
```

Comment:
Could you add a note to the note top of this file similar to the one over in `dask/distributed`? 

https://github.com/dask/distributed/blob/1314ebb8671493290d284bf24b1ed1da1749f90c/.pre-commit-config.yaml#L1-L2

Reply:
Sure, added